src/Diagnostics.DataProviders/DataProviders/Base/SupportObserverDataProviderBase.cs

```diff
@@ -25,45 +26,61 @@ public SupportObserverDataProviderBase(OperationDataCache cache, SupportObserver
             FillObserverRouteTemplates();
         }
 
-        public async Task<dynamic> GetResource(string observerUrl)
+        public Task<dynamic> GetResource(string resourceUrl)
         {
             Uri uri;
-            Dictionary<string, string> routeParametersAndValues = null;
-            string routeTemplate = null;
+
+            var allowedHosts = new string[] { "wawsobserver.azurewebsites.windows.net", "support-bay-api.azurewebsites.net", "support-bay-api-stage.azurewebsites.net" };
 
             try
             {
-                uri = new Uri(observerUrl);
+                uri = new Uri(resourceUrl);
 
-                if (!uri.Host.Equals("wawsobserver.azurewebsites.windows.net", StringComparison.CurrentCultureIgnoreCase))
+                if (!allowedHosts.Any(h => uri.Host.Equals(h, StringComparison.CurrentCultureIgnoreCase)))
                 {
-                    throw new FormatException($"{observerUrl} is not for an Observer call. Please use a URL that points to Observer. Eg., https://wawsobserver.azurewebsites.windows.net/Sites/mySite");
+                    throw new FormatException($"Cannot make a call to {uri.Host}. Please use a URL that points to one of the hosts: {string.Join(',', allowedHosts)}");
                 }
             }
             catch (ArgumentNullException)
             {
-                throw new ArgumentNullException("observerUrl");
+                throw new ArgumentNullException("resourceUrl");
             }
             catch (UriFormatException ex)
             {
                 string exceptionMessage = null;
 
                 if (ex.Message.Contains("The URI is empty"))
                 {
-                    exceptionMessage = "ObserverUrl is empty. Please pass a non empty string for observerUrl";
+                    exceptionMessage = "ResourceUrl is empty. Please pass a non empty string for resourceUrl";
                 }
 
                 //fix for travis ci
-                if (!observerUrl.StartsWith("https://wawsobserver.azurewebsites.windows.net") || !observerUrl.StartsWith("http://wawsobserver.azurewebsites.windows.net"))
+                if (!resourceUrl.StartsWith("https://wawsobserver.azurewebsites.windows.net") || !resourceUrl.StartsWith("http://wawsobserver.azurewebsites.windows.net"))
```

Comment:
Do you need this line still?

Reply:
Looks like this was a copy-paste without editing. Will fix